VeganVirus/VeganVirus/ImageAction.cpp

```diff
@@ -0,0 +1,93 @@
+#include "ImageAction.h"
+
+
+
```

Comment:
I prefer to write comments inside the header files, not in the .cpp implementation files.
Also, you have multiple consecutive empty lines, which I think you should remove.

Reply:
I still think the comments should be in the header files, to be consistent with all other classes which have already been implemented.